scripts/create-clusters.sh

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script handles the creation of multiple clusters using kind and the
+# ability to create and configure an insecure container registry.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source "$(dirname "${BASH_SOURCE}")/util.sh"
+CONFIGURE_INSECURE_REGISTRY="${CONFIGURE_INSECURE_REGISTRY:-}"
+CONTAINER_REGISTRY_HOST="${CONTAINER_REGISTRY_HOST:-172.17.0.1:5000}"
+NUM_CLUSTERS="${NUM_CLUSTERS:-2}"
+docker_daemon_config="/etc/docker/daemon.json"
+
+function create-and-configure-insecure-registry() {
+  # Run insecure registry as container
+  docker run -d -p 5000:5000 --restart=always --name registry registry:2
```

Comment:
(no action required) This could conflict with user environment, as the user might already be running a registry. would it be good if we make this configurable and optional?

Reply:
I think that's a good suggestion that can be implemented in a follow-up.